src/main/java/seedu/address/logic/commands/AddNumberOfRoomsCommand.java

```diff
@@ -14,26 +11,33 @@
 public class AddNumberOfRoomsCommand extends Command {
     public static final String COMMAND_WORD = "addnumberofrooms";
     private int numOfRooms;
```

Comment:
Good to include an attribute string for message usage (similar to the other commands)